docs/get-started/getting-started-azure-synapse-analytics.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Azure Synapse Analytics
+nav_order: 5
+parent: Getting-Started
+---
+# Getting started with RAPIDS Accelerator on Azure Synapse Analytics
+ [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is a limitless analytics service that brings
```

Comment:
maybe defer the superlatives such as "limitless" to the documentation hosted externally?

Reply:
Removed "limitless". @gerashegalov